include/lcp-catlist.php

```diff
@@ -447,6 +460,52 @@ public function get_thumbnail($single, $lcp_thumb_class = null){
       $lcp_thumb_class);
   }
 
+  public function get_outer_tag($tag, $css_class) {
+    $css_class = $this->params['class'] ?: $css_class;
+
+    $tag_string = '<' . $tag;
+    if ($tag == 'ol' && !empty($this->params['ol_offset'])) {
+      $this->lcp_output .= ' start=' . $this->params['ol_offset'];
+    }
+
+    // Follow the numner of posts in an ordered list with pagination
+    if( 'ol' === $tag && $this->page > 1 ){
+      $start = $this->params['number_posts'] * ( $this->page - 1 ) + 1;
```

Comment:
Strange, no idea why I changed that line ¯\_(ツ)_/¯ Originally, as it is now, it was:
 https://github.com/picandocodigo/List-Category-Posts/blob/35bd4409e173adfe1936bde6e03fc85cbac177d7/include/lcp-catlistdisplayer.php#L120

so probably thought "hmm, I'm moving it to the Catlist class so maybe I'll also remove the getter method call" (and make a mistake 😆 ) but maybe better to leave it more or less as it was?
```php
$start = $this->get_number_posts() * ($this->catlist->get_page() - 1) + 1;
```
I will also add a test case for this because clearly pagination tests didn't catch it.

Reply:
Fixed in 238c548 together with a few other fixes.